.changelogs/README.md

```diff
@@ -0,0 +1,54 @@
+# Changelogs
+
+This directory includes temporary changelog entries, in the form of simple `.json` files. This was done to prevent merge conflicts when modifying the same `CHANGELOG.md` file in many PR's at once.
+
+
+## Mandatory PR check
+
+Every pull request in this repository requires a new changelog entry to be created in this directory, asserted by a GitHub actions workflow. The workflow will fail in any PR that does not have a new `.changelogs/*.json` file added. If a pushed commit does not have a PR associated with it, the check is skipped entirely.
+
+**To disable this check**, simply add the following string to the **PR description**:
+
+```
+[skip changelog check]
```

Comment:
Sure, the shorter string looks better https://github.com/handsontable/handsontable/pull/7448/commits/b2fbf6c73a0cbc5d2d31be1de02892a1081783b4.

I added some more information about when the check runs that should hopefully be more clear. I'm keeping it on `push` only (as opposed to `push` and *edit pr description* events) as it would create a lot of noise (and potentially friction) when editing the description.

I would personally prefer to keep the `skip`'s in the description rather than the title (what if we start adding more, similar strings?), as you already get an error associated with a commit that should be pretty visible on its own.

Reply:
That makes sense, thx!